layers/meta-balena-raspberrypi/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -16,3 +16,11 @@ SRC_URI += " \
     file://rpi.h-Remove-usb-start-from-CONFIG_PREBOOT.patch \
     file://0002-raspberrypi-Disable-simple-framebuffer-support.patch \
 "
+
+# Disable flasher check since it starts usb unnecessarily
+# and we don't generate flasher images for any of the RPIs.
+SRC_URI_append = " \
+    file://0001-rpi-Disable-image-flasher-check.patch \
+"
+
+RESIN_UBOOT_DEVICE_TYPES_append_raspberrypi3-64 = " usb"
```

Comment:
Why add it only for 1 machine?

Reply:
@floion the issue was reported for pi3, but as I mentioned in the PR message above, we can of-course add for all of them.

Reply:
@petrosagg shall we add this for all the pi family?

Reply:
@acostach let's add it for all the pis